tests/Kernel/fixtures/patterns/organisation_teaser_rendering.yml

```diff
@@ -0,0 +1,56 @@
+- fields:
+    logo:
+      src: "https://placeimg.com/100/100"
+      alt: "Alternative text for logo"
+    name: "Organisation name"
+    acronym: "Acronym"
+    organisation_details:
+      - label: "Address"
+        body: "61 Lorem Ipsum Street, Brussels, Belgium, 1000"
+      - label: "Website"
+        body: '<a href="http://www.example.com">http://www.example.com</a>'
+    contact_page_url: "http://www.example.com/contact"
+  assertions:
+    count:
+      div[role="img"].ecl-u-d-lg-block: 1
+      a.ecl-link[href="http://www.example.com/contact"]: 1
+    equals:
+      h4.ecl-u-type-heading-4: "Organisation name | Acronym"
```

Comment:
We also need to test that, when acronym is not there, then no ` | ` will be shown.

Reply:
Next text checks it.

Reply:
Next test checks it.